packages/mdc-select/foundation.js

```diff
@@ -219,9 +257,13 @@ export default class MDCSelectFoundation extends MDCFoundation {
     this.adapter_.setMenuElStyle('transform-origin', `center ${itemOffsetTop}px`);
   }
 
-  close_() {
+  close_(shouldRemoveFloatingLabelClass) {
```

Comment:
Well, I know what my next Medium post is going to be. hahaha

Reply:
So, the only argument I have is that this example doesn't really fall into what's described in that blog post. Since I define a variable and pass that, it's entirely readable and easy to assume what the boolean is governing in the `close_()` method. Thoughts?

Reply:
That's a very good point - and thanks for using such explicit variable names 😄  🎉 

Personally, I would still prefer an enum because it prevents an entire class of human errors by forcing the caller to use statically typed values with descriptive names. It also looks a little cleaner IMO.

E.g.:

```js
const shouldRemoveFloatingLabelClass = true;
this.close_(shouldRemoveFloatingLabelClass);
```

vs.

```js
this.close_(FloatingLabelBehavior.REMOVE_ON_CLOSE);
```

In this particular case I don't feel _too_ strongly either way, though, since it's a private method with only one param. So it's totally up to you 🙂

Reply:
So, I'd prefer avoiding the enum, but before I even weigh in on that conversation... do we even need this parameter?

We're calling this method in 2 places: one place always passes `true`, while the other always passes `false`. Can't we just move the remaining condition and `removeClassFromLabel` call to the location where we're currently calling this with `true`?

Reply:
Shooooot. Ken wins. The only caveat being we have to check if an item is selected in the cancel handler before removing that label.